bundled-with-dotnet-single-file-deployment.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: bundled with .NET single-file deployment
+    namespace: runtime/dotnet
+    authors:
+      - sara.rincon@mandiant.com
+    scope: file
+    references:
+      - https://learn.microsoft.com/en-us/dotnet/core/deploying/single-file/overview?tabs=cli
+      - https://github.com/dotnet/runtime/blob/84de9b678613675e0444b265905c82d33dae33a8/src/installer/managed/Microsoft.NET.HostModel/AppHost/HostWriter.cs
+    examples:
+      - a83339b07cf2bf1aeda192de42760c625d4b2f106260f58c902d02f4766848d5
+  features:
+    - or:
+      - and:
+        - match: contains PDB path
+        - string: "singlefilehost.pdb"
```

Comment:
can you add a description why this may be a good signal?

Reply:
sure.  this is the symbol file for the native executable when publish single file is set to True